README.md

```diff
@@ -62,9 +66,12 @@ cargo install --git https://github.com/wert007/commit-analyzer
 ```
 
 This command will install a release compilation of the latest version locally to
-your user account.
+your user account. The same command also works for regular updates, this is, new
+commits have had been introduced since the last installation.
 
-In order to install the latest updates, rerun with `--force`:
+Cargo will refuse to replace the currently installed executable if no new
+commits were found. To override the current binary with the latest repository 
+state anyway, rerun with `--force`:
```

Comment:
Question: When would one need to do that?

Reply:
Diverging or broken histories as well as enforced overwrites.

If one of us should squash the `main` branch for some reasons, the history will differ from the repository state that Cargo downloaded when the last installation was requested.  Cargo might be confused about the differing Git histories and might, hence, refuse to install.

Another point is the forced update.  Sometimes, you just want to install the latest version and do not care about details such as whether the respective repositories changed at all.  Imagine you design an update script for all of your Cargo maintained applications.  You would risk to break the workflow if one update should fail for some reason.  That is when forced updates come in handy.

Reply:
> Diverging or broken histories as well as enforced overwrites.
> 
> If one of us should squash the `main` branch for some reasons, the history will differ from the repository state that Cargo downloaded when the last installation was requested. Cargo might be confused about the differing Git histories and might, hence, refuse to install.

I suspect, that cargo will tell, if this happens, and what to do to fix this.

> Another point is the forced update. Sometimes, you just want to install the latest version and do not care about details such as whether the respective repositories changed at all. Imagine you design an update script for all of your Cargo maintained applications. You would risk to break the workflow if one update should fail for some reason. That is when forced updates come in handy.

This is also not relevant for this tool, I think?

Reply:
Complete removal of this paragraph? :wrench: